src/trix/config/input.js

```diff
@@ -3,6 +3,8 @@ import { makeElement, removeNode } from "trix/core/helpers/dom"
 
 const input = {
   level2Enabled: true,
+  fileInputId: `trix-file-input-${Date.now().toString(16)}`,
+  acceptedFileTypes: '*',
```

Comment:
We prefer double quotes for strings. The linter should flag this.

Reply:
Fixed.